README.md

```diff
@@ -20,14 +22,16 @@ This repository contains two `docker-compose` configuration files:
 
 ### Development (databases and Keycloak)
 
+Configure the environment variables in the dev compose file (e.g. keycloak host).
```

Comment:
Not directly related to this PR, but we should place all variables inside a `.env` file.

Reply:
See !15.